apps/system/lockscreen/js/charging.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+(function(exports) {
+
+/**
+ * Creates an object used for refreshing the charging string UI element.
+ * @class  ChargingStatus
+ */
+function ChargingStatus() {
```

Comment:
Sadly we don't have LockScreen app yet so every file and component of LockScreen should be prefixed with 'LockScreen' to prevent name collision under System app. ex: `LockScreenChargingStatus`